plugins/scaffolder-backend/src/lib/templating/SecureTemplater.ts

```diff
@@ -109,6 +112,7 @@ export interface SecureTemplaterOptions {
   templateFilters?: Record<string, TemplateFilter>;
   /* Extra user-provided nunjucks globals */
   templateGlobals?: Record<string, TemplateGlobal>;
+  nunjucksConfigs?: NunjucksConfigs;
```

Comment:
can we just inline the type here rather than exporting a new one?

Reply:
Don't think we need to export the type actually in the first place as the `SecureTemplater` isn't exported.